app/views/pages/tutorials/pages/pages.liquid

```diff
@@ -41,7 +55,7 @@ layout_name: my_custom_layout
 | redirect_to            | URL to a page you want to redirect                           |                                                              |               | No       |
 | response_headers       | JSON object that you can define to override most of the http headers |                                                              |               | No       |
 | searchable             | Indication that this page should be indexed and accessed via GraphQL query | true                                                         | false         | No       |
-| slug                   | Defines the URL at which this page will be accessible. Assuming your site domain is https://example.com, you will be able to access the page at `https://example.com/[slug]` |                                                              | n/a           | Yes      |
+| slug                   | Use the `slug` property to overwrite the default  slug.   |                                                              | The default slug is automatically created based on the page's location in the codebase. For example, a page `app/views/pages/get-started/hello-world.liquid` has the default slug `get-started/hello-world`. | No      |
```

Comment:
I think this will render incorrectly. Long desc. will land in options column

Reply:
It looks correct on my staging: https://diana-documentation.staging.oregon.platform-os.com/tutorials/pages/pages